app/containers/scenes/MainMapScene.js

```diff
@@ -166,18 +166,20 @@ export default class MainMapScene extends Component {
 	}
 
 	_createTrailingMarkersForChallenge(challenge) {
-		var trailingMarkers = challenge.locations.slice(1).map((location, index) => (
-			{
-				id: challenge.id + "-" + (index+1),
-				title: location.title,
-				chalengeDescription: challenge.description,
-				latlng: {
-					latitude: location.latitude,
-					longitude: location.longitude
-				},
-			}
-		));
-		return trailingMarkers;
+		if(challenge.locations && challenge.locations.length > 1) {
```

Comment:
@jwt1143 fair point. I think I changed this trying to fix the error I mentioned above. Challenge was getting passed in as `undefined` so I was trying to guard against that. I could have done it a lot cleaner, and the real problem exists outside this function (as stated above) so we can probably just toss this.